ohapi/tests/test_api.py

```diff
@@ -174,7 +177,6 @@ class APITestMessage(TestCase):
     def setUp(self):
         pass
 
-    @my_vcr.use_cassette()
```

Comment:
Removing this line is what is causing the test failure - it's not using any cassette to test against :)